controls/radvirtualgrid/mvvm.md

```diff
@@ -0,0 +1,327 @@
+---
+title: MVVM Support
+page_title: MVVM Support
+description: MVVM Support
+slug: virtualgrid-mvvm
+tags: virtualgrid,mvvm
+published: True
+position: 1
+---
+
+# MVVM Support
+
+This topic will go through the approach of using __RadVirtualGrid__ in an MVVM environment. This can be achieved by benefiting from the __DataProvider__ mechanism of the control. More information on this matter can be found in the [Getting Started]({%slug virtualgrid-getting-started2%}) and [Custom DataProvider]({%slug virtualgrid-custom-dataprovider%}) topics.
+
+## Defining the business model
+
+Firstly, we need an object model that will be used for further populating __RadVirtualGrid__ with data. For this purpose, the following __Club__ object is defined.
+
+#### __[C#] Example 1: Defining the Club object__
+
+{{region radvirtualgrid-mvvm_0}}
+	public class Club : INotifyPropertyChanged
```

Comment:
Can we use ViewModelBase here? This will shorten the code snippet. The implementation of the PropertyChanged event can be removed.